OrbitGl/App.cpp

```diff
@@ -571,22 +567,23 @@ bool OrbitApp::StartCapture() {
   int32_t pid = Capture::GTargetProcess->GetID();
   std::map<uint64_t, FunctionInfo*> selected_functions =
       Capture::GSelectedFunctionsMap;
-  thread_pool_->Schedule([this, pid, selected_functions] {
-    capture_client_->Capture(pid, selected_functions);
-  });
+
+  result = capture_client_->StartCapture(thread_pool_.get(), pid,
```

Comment:
You are getting the result but not checking if it contains error message afterwards

Reply:
indeed - good catch, thanks!

Reply:
done.